src/Components/styles/LandingPage.css

```diff
@@ -62,13 +62,35 @@
     width: 70%;
 }
 
+#section-3-h2 {
+    color: #190078;
+    font-weight: 600;
+    font-size: 40px;
+    width: 100%;
+    float: right;
+}
+
+#section-4-h2 {
+    color: #190078;
+    font-weight: 600;
+    font-size: 40px;
```

Comment:
Why px? What if the screen is high res? Is there a pixel density independent way to accomplish this for text scaling??? @zhengwin

Reply:
yea we should use rem instead of px. Rem scales relative to the html size

Reply:
changed the font-sizes to `rem`. should every instance of `px` be `rem`? (should there be no `px` at all?)

Reply:
I do think every instance of px should be something pixel density independent (ie rem). @zhengwin would you agree or can u think of edge cases to use px?